tests/xlsx_export.test.ts

```diff
@@ -832,37 +837,40 @@ describe("Test XLSX export", () => {
           },
         ],
       });
-      createChart(
-        model,
-        {
-          dataSets: ["Sheet1!B1:B9"],
-          labelRange: "Sheet1!A2:A9",
-          aggregated: true,
-          type: "bar",
-        },
-        "1"
-      );
-      createChart(
-        model,
-        {
-          dataSets: ["Sheet1!B1:B9"],
-          labelRange: "Sheet1!A2:A9",
-          aggregated: true,
-          type: "line",
-        },
-        "1"
-      );
-      createChart(
-        model,
-        {
-          dataSets: ["Sheet1!B1:B9"],
-          labelRange: "Sheet1!A2:A9",
-          aggregated: true,
-          type: "pie",
-        },
-        "1"
-      );
+      for (const type of ["bar", "line", "pie"]) {
+        createChart(
+          model,
+          {
+            dataSets: ["Sheet1!B1:B9"],
+            labelRange: "Sheet1!A2:A9",
+            aggregated: true,
+            type: type as "bar" | "line" | "pie",
+          },
+          "1"
```

Comment:
the loop is kinda useless if we create 3 times a chart with the same id  so we end up with only one chart :upside_down_face:

Reply:
Totally agree but the process was the same before (creating 3 charts with the same Id), I juste made it shorter. Maybe it should be a better idea to split it in 3 tests